content/api/commands/intercept.md

```diff
@@ -993,25 +1040,34 @@ By calling `req.on`, you can subscribe to different events:
 cy.intercept('/shop', (req) => {
   req.on('before:response', (res) => {
     /**
-     * Emitted before `response` and before any `req.continue` handlers.
-     * Modifications to `res` will be applied to the incoming response.
-     * If a promise is returned, it will be awaited before processing other event handlers.
+     * Emitted before `response` and before any `req.continue`
+     * handlers.
```

Comment:
Would this read better if the lines were formatted into a single paragraph, instead of 3 separate wrapping lines?

Reply:
(same for the following comment blocks)

Reply:
Done